### DISCUSSION TITLE
My App Defaults - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#7] [Announcements]
# My App Defaults - 1nan

https://yinan.me/my-defaults

**Comment by andrewcanion** [comment]
Awesome to see another DevonThink user!

**Reply by yinan-c** [reply]
Nice I love Devonthink, such a great tool.

**Comment by yinan-c** [comment]
Beside this list, I’d like to list some of my favorite apps that were not mentioned:

- Launcher: [Alfred](https://alfredapp.com).
- Window management: [Swish](https://highlyopinionated.co/swish/), [wins](https://wins.cool), [moom](https://manytricks.com/moom/)
- App store: [SetApp](https://setapp.com/)
- Time management/pomodoro app: [focus](https://heyfocus.com), [Session](https://stayinsession.com/), [Timing](https://timing.app)
- Book management: [Book Tracker](https://apps.apple.com/gb/app/book-tracker-bookshelf-log/id1491660771), [Calibre](https://calibre-ebook.com/)
- Reference management: formally [Mendeley](https://mendeley.com/), now [Zotero](https://www.zotero.org/)
- Input source related: [Squirrel](https://github.com/rime/squirrel), [Switchkey](https://github.com/itsuhane/SwitchKey), [Showyedge](https://showyedge.pqrs.org/), [Input Source Pro](https://inputsource.pro/)
- Translate: [Easydict](https://github.com/tisfeng/Easydict), [PopClip](https://www.popclip.app/), Alfred
- Downtime management: [Sofa](https://apps.apple.com/gb/app/sofa-downtime-organizer/id1276554886), [Movie Tracker](https://apps.apple.com/gb/app/movie-tracker-what-to-watch/id6443877426)
- Subscription management: [Subscriptions](https://apps.apple.com/gb/app/subscriptions-track-expenses/id1577082754)
- Automation: [Keyboard Maestro](http://www.keyboardmaestro.com/), Alfred, Shortcuts, and Applescripts
- Other apps: Popclip, [SearchLink](https://brettterpstra.com/projects/searchlink/), [Hookmark](https://hookproductivity.com/), [Homerow](https://www.homerow.app/), [neovim](https://neovim.io/), [Foxtrot pro](http://foxtrot-search.com/)

I use these apps extensively and benefit a lot from them, but they don’t fit into the original categories. Now that the list is quite long, I will probably write another blog post listing my favorite apps/tools someday.

**Comment by miantiao-me** [comment]
iCloud 订阅了么？ iCloud 邮箱有没有用自定义域名

**Reply by yinan-c** [reply]
订阅了 Apple One 带着 iCloud 2TB plan，然后没有用自定义域名。
